### ISSUE TITLE
feedback missing when (star) rating is submitted

### ISSUE THREAD

**Issue by anasereijo** [ubuntu-flutter-community/software#1179]
Currently, there is no feedback telling the rating was saved/submitted.

https://user-images.githubusercontent.com/36884067/226974069-c0a0a689-1fc4-4e7d-a773-28217821c69a.mov

It would be nice to display a message telling the user the rating was submitted successfully.

wireframe:
<img width="1333" alt="Screenshot 2023-03-22 at 16 36 02" src="https://user-images.githubusercontent.com/36884067/226974743-88c076ab-11b9-485b-9715-d5e6158d26df.png">

**Comment by anasereijo** [comment]
Actually, we should maybe also have a way to undo the action. Is this possible?

amazon example

<img width="742" alt="Screenshot 2023-03-22 at 16 43 40" src="https://user-images.githubusercontent.com/36884067/226977272-15e734be-9d08-4416-90c7-edf5ac55113c.png">

**Comment by Feichtmeier** [comment]
Good question. Would need to test this. Looking at what the odrs.dart lib does is sending a HTTP POST for creating a review.
HTTP apis normally then also have a HTTP PUT method implementation for this http resource, with given id and changed values. But this depends on what the odrs system does on the other side

**Comment by anasereijo** [comment]
> on the feedback, would you like a small "toast" 🍞 popping up at the bottom? (without 🧈 )

Yes, that would be great and consistent with what we do elsewhere.
can we have the following text? "Thank you for your feedback!" with the link to undo

**Comment by anasereijo** [comment]
> I must admit that when I really put on the user glasses and stare at this row of stars and the button... isn't this actually very confusing? Click to rate.... write your review.... why don't we only have the review button? Which opens the dialogue with both stars and review?

I think I understand your point, but I don't think it is _very_ confusing. The reasoning is that some users might prefer to _just_ quickly **Rate** the package without further details, and some users might want to provide more details with a **Review**. Surely we could have it all in one button with the dialogue and make reviews optional, but I believe that it is more misleading than having both options available.

I also checked the notes from the latest round of user interviews, and seems that no issues were flagged on this. Doesn't necessarily mean it's perfect, but I wouldn't change it just now.

This is all open for discussion, and I will be looking at the reviews/ratings soon as we need to have a way for users to edit their reviews (as discussed last week), but I will keep it like this for now.

**Comment by anasereijo** [comment]
Forgot to say 'thank youuu for your input' I am sorry :)

**Comment by anasereijo** [comment]
Sadly, I have to close this, as it's not possible with ODRS. Hopefully it will be possible at some point in the future.
